x/bep3/types/genesis_test.go

```diff
@@ -41,74 +40,50 @@ func (suite *GenesisTestSuite) TestValidate() {
 		{
 			"default",
 			args{
-				swaps:    types.AtomicSwaps{},
-				supplies: types.AssetSupplies{},
```

Comment:
Shouldn't we add `supplies` back into the args and tests now that it's been readded to types/genesis.go?

Reply:
addressed in #623